lesson_3/models.py

```diff
@@ -0,0 +1,30 @@
+from flask_sqlalchemy import SQLAlchemy
+from flask_wtf import FlaskForm
+from wtforms import StringField, PasswordField
+from wtforms.validators import DataRequired, Email, EqualTo, Length
+
+db = SQLAlchemy()
+
+
+class User(db.Model):
+    id = db.Column(db.Integer, primary_key=True)
+    first_name = db.Column(db.String(50), nullable=False)
+    last_name = db.Column(db.String(50), nullable=False)
+    email = db.Column(db.String(120), unique=True, nullable=False)
+    password = db.Column(db.String(100), nullable=False)
+
+
+class RegistrationForm(FlaskForm):
+    first_name = StringField('Имя', validators=[DataRequired()])
+    last_name = StringField('Фамилия', validators=[DataRequired()])
+    email = StringField('Email', validators=[DataRequired(), Email()])
+    password = PasswordField('Пароль', validators=[DataRequired(), Length(min=6),
+                                                   EqualTo('confirm_password',
+                                                   message='Пароли должны совпадать')])
+    confirm_password = PasswordField('Повторите пароль', validators=[DataRequired()])
+
```

Comment:
Если добавить поле `SubmitField`
```suggestion
    submit = SubmitField('Зарегистрироваться')
```